extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -59,15 +59,24 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
           // and the signature is "FAKESIG". This mode is only allowed in
           // `localDev` and primarily used for A4A Envelope for testing.
           // See DEVELOPING.md for more info.
-          const creative = this.transformCreativeLocalDev_(deserialized);
```

Comment:
Ignore changes to this file. Merge conflict with https://github.com/ampproject/amphtml/pull/9331.